marvin/team.py

```diff
@@ -129,6 +129,10 @@ async def control_function(gh: gh_aiohttp.GitHubAPI, token: str) -> bool:
     ActivityLimitedReviewer(gh_name="lovesegfault", days=3, limit=1),
     ActivityLimitedReviewer(gh_name="lovesegfault", days=7, limit=7, can_merge=True),
     ActivityLimitedReviewer(gh_name="thiagokokada", days=7, limit=5),
+    ActivityLimitedReviewer(gh_name="supersandro2000", days=1, limit=25),
+    ActivityLimitedReviewer(
+        gh_name="supersandro2000", days=1, limit=25, can_merge=True
+    ),
```

Comment:
Now it is added twice?

Reply:
Three lines above lovesegfault is also added two times. I am not sure if this is because the different limit or because of can_merge. Did not look to deep into this.

Reply:
indeed.

Reply:
Yes, "reviewers" and "mergers" are currently treated as two separate sets. You need to add yourself to both if you want to get review requests for `needs_reviewer` and `needs_merger` MRs. That way you can specify two different limits.